src/AkeneoPimClientBuilder.php

```diff
@@ -214,4 +198,32 @@ public function getStreamFactory()
 
         return $this->streamFactory;
     }
+
+    /**
+     * @param Authentication $authentication
+     *
+     * @return array
+     */
+    protected function setUp(Authentication $authentication)
```

Comment:
private

Reply:
No, it is called in the EE client builder too.